modules/tinymce/src/plugins/quickbars/test/ts/browser/KeyboardShortcutContextualMenuTest.ts

```diff
@@ -0,0 +1,33 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyHooks, TinySelections, TinyContentActions, McEditor, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import QuickbarsPlugin from 'tinymce/plugins/quickbars/Plugin';
+
+describe('browser.tinymce.core.focus.KeyboardShortcutContextualMenuTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    plugins: 'quickbars',
+    quickbars_insert_toolbar: 'quicktable image media codesample',
+    quickbars_selection_toolbar: 'bold italic underline | formatselect | bullist numlist | blockquote quicklink',
+    contextmenu: 'undo redo | inserttable | cell row column deletetable | help',
+    base_url: '/project/tinymce/js/tinymce',
+  }, [ QuickbarsPlugin ]);
```

Comment:
```suggestion
  }, [ QuickbarsPlugin ], true);
```
This is how we make sure the editor is focused. If it needs to be done per test instead of just at the start then we should use a `beforeEach` hook instead.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/6f20447b960bd398b43e96efc00dbad2dd656b76